launch/tier4_perception_launch/launch/object_recognition/detection/detection.launch.xml

```diff
@@ -211,6 +216,29 @@
       <arg name="radar_object_clustering_param_path" value="$(var radar_object_clustering_param_path)"/>
     </include>
   </group>
+  <group if="$(var switch/detector/camera_bev)">
```

Comment:
IMHO I think you can do 
```suggestion
  <group if="$(eval '&quot;$(var mode)&quot;==&quot;camera&quot;')">
```
instead and remove `switch/detector/camera_bev` arg to make the launcher as simple as possible, since we don't have any other choice for `mode=camera`.

Reply:
At first, I did write it this way, only judging whether to call my node based on `perception_mode`. However, when I submitted the PR, I found that the launch style of the perception module had completely changed, and I had to adjust my code to adapt to the new style.

Now, I  I will modify according to your suggestion to make the launcher as simple as possible.